includes/class.llms.review.php

```diff
@@ -42,58 +43,82 @@ public function __construct() {
 			?>
 			<div id="old_reviews">
-			<h3><?php echo apply_filters( 'lifterlms_reviews_section_title', __( 'What Others Have Said', 'lifterlms' ) ); ?></h3>
+			<h3><?php echo esc_html( apply_filters( 'lifterlms_reviews_section_title', __( 'What Others Have Said', 'lifterlms' ) ) ); ?></h3>
 			<?php
-			$args        = array(
+			$args = array(
 				'posts_per_page'   => get_post_meta( get_the_ID(), '_llms_num_reviews', true ),
 				'post_type'        => 'llms_review',
 				'post_status'      => 'publish',
 				'post_parent'      => get_the_ID(),
 				'suppress_filters' => true,
 			);
+
 			$posts_array = get_posts( $args );
 
-			$styles = array(
-				'background-color' => '#EFEFEF',
-				'title-color'      => 'inherit',
-				'text-color'       => 'inherit',
-				'custom-css'       => '',
+			$styles = apply_filters(
+				'llms_review_custom_styles',
+				array(
+					'background-color' => '#EFEFEF',
+					'title-color'      => 'inherit',
+					'text-color'       => 'inherit',
+					'custom-css'       => '',
+				)
 			);
 
-			if ( has_filter( 'llms_review_custom_styles' ) ) {
-				$styles = apply_filters( 'llms_review_custom_styles', $styles );
+			$inline_styles = '';
+
+			if ( $styles['background-color'] ?? '' ) {
+				$inline_styles .= '.llms_review{background-color:' . $styles['background-color'] . '}';
 			}
 
-			foreach ( $posts_array as $post ) {
-				echo $styles['custom-css'];
+			if ( $styles['title-color'] ?? '' ) {
+				$inline_styles .= '.llms_review h5{color:' . $styles['title-color'] . '}';
+			}
+
+			if ( $styles['text-color'] ?? '' ) {
+				$inline_styles .= '.llms_review h6,.llms_review p{color:' . $styles['text-color'] . '}';
+			}
 
+			if ( $styles['custom-css'] ?? '' ) {
+
+				// Remove style tags in case they were added with the filter.
+				$inline_styles .= str_replace( array( '<style>', '</style>' ), '', $styles['custom-css'] );
+			}
+
+			if ( $inline_styles ) {
+				echo '<style id="llms_review_custom_styles">' . $inline_styles . '</style>';
+			}
+
+			foreach ( $posts_array as $post ) {
 				?>
-				<div class="llms_review" style="margin:20px 0px; background-color:<?php echo $styles['background-color']; ?>; padding:10px">
-					<h5 style="font-size:17px; color:<?php echo $styles['title-color']; ?>;" style="margin:3px 0px"><strong><?php echo get_the_title( $post->ID ); ?></strong></h5>
-					<h6 style="font-size:13px; color:<?php echo $styles['text-color']; ?>;"><?php echo sprintf( __( 'By: %s', 'lifterlms' ), get_the_author_meta( 'display_name', get_post_field( 'post_author', $post->ID ) ) ); ?></h6>
-					<p style="font-size:15px; color:<?php echo $styles['text-color']; ?>;"><?php echo get_post_field( 'post_content', $post->ID ); ?></p>
+				<div class="llms_review">
+					<h5><strong><?php echo get_the_title( $post->ID ); ?></strong></h5>
+					<h6><?php echo esc_html( __( 'By: ', 'lifterlms' ) . get_the_author_meta( 'display_name', get_post_field( 'post_author', $post->ID ) ) ); ?></h6>
```

Comment:
I'd prefer not to change the localized string here, so it doesn't require an update, is there a reason why you prefer string concatenation over `sprintf`?
Also I'd consider that while something like "By: author" makes perfectly sense it might be not that obvious for some locales. See https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#best-practices-for-writing-strings 

```suggestion
					<h6>
					<?php
						// Translators: %s = The author display name.
						echo esc_html( sprintf( __( 'By: %s', 'lifterlms' ), get_the_author_meta( 'display_name', get_post_field( 'post_author', $post->ID ) ) ) );
					?>
					</h6>
```